scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java

```diff
@@ -499,6 +502,24 @@ protected void buildCellContextualMenu(Menu menu) {
                       openItem,
                       exportXmlItem,
                       removeItem);
+
+        controller.getJobSignals(ids.get(0), menu, this);
+
+    }
+
+    public void addActionsMenu(String jobId, Menu menu, Set<String> signals) {
+        MenuItem actionsItem = new MenuItem("Actions");
+        Menu signalsMenu = new Menu();
+        for (final String signal : signals) {
+            MenuItem item = new MenuItem(signal.substring(signal.indexOf("_") + 1));
+            item.addClickHandler(event2 -> controller.addJobSignal(signal.substring(signal.indexOf("_") + 1), jobId));
```

Comment:
event2 is never used

Reply:
if it is really needed, let's change event2 to just event

Reply:
I will rename it